README.md

```diff
@@ -479,6 +479,10 @@ The services included with this setup rely on environment variables to work prop
 | SHARED_FOLDER_ENABLED | `string` | Specifies the use of shared folder (between grader and student notebooks)  | `True` |
 | SPAWNER_MEM_LIMIT | `string` | Spawner memory limit | `2G` |
 | SPAWNER_CPU_LIMIT | `string` | Spawner cpu limit | `0.5` |
+| POSTGRES_NB_HOST | `string` | Postgres host for Nbgrader |  |
```

Comment:
Actually, can't we just use a generic connection string since we use SQLAlchemy? For example:
```
    SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL') or \
        'sqlite:///' + os.path.join(<path-to-course-folder>, 'gradebook.db')
```

Reply:
yeah, we can replace all the current env-vars related with postgres and use a single value contained the url

Reply:
btw, I just made this change/refactor in a git stash if you would like to go with a single ENV var ( NBGRADER_DATABASE_UR )

Reply:
Great, thanks for clarifying, that makes perfect sense.